pyk/src/pyk/cterm/cterm.py

```diff
@@ -187,7 +187,7 @@ def add_constraint(self, new_constraint: KInner) -> CTerm:
         return CTerm(self.config, [new_constraint] + list(self.constraints))
 
     def anti_unify(
-        self, other: CTerm, keep_values: bool = False, kdef: KDefinition | None = None
+        self: CTerm, other: CTerm, keep_values: bool = False, kdef: KDefinition | None = None
```

Comment:
```suggestion
        self, other: CTerm, keep_values: bool = False, kdef: KDefinition | None = None
```

To commit after CI finishes.